app/controllers/myott/subscriptions_controller.rb

```diff
@@ -4,7 +4,12 @@ class SubscriptionsController < MyottController
     before_action :authenticate, except: %i[start invalid]
 
     def start; end
-    def invalid; end
+
+    def invalid
+      unless current_user.nil?
```

Comment:
Personal preference, but I find the double negative confusing. Up to you though.
```suggestion
      if current_user.present?
```

Reply:
no worries!